### ISSUE TITLE
Uncaught TypeError: Cannot read properties of undefined (reading '__H')

### ISSUE THREAD

**Issue by nmocruz** [preactjs/preact-cli#1729]

**Comment by nmocruz** [comment]
I updated the link,

**Comment by rschristian** [comment]
Is it a Preact-CLI issue? You have various Webpack dependencies and a `webpack.config.js` sitting there.

It's unclear whether you're using Webpack directly or using Preact-CLI.

**Comment by nmocruz** [comment]
I dont thing is Preact-CLI, I am using 
```
npx webpack serve
```

**Comment by JoviDeCroock** [comment]
Heh, sorry hence the ask for reproduction steps 😅 I saw preact-cli in the package json and that worked fine for me

**Comment by rschristian** [comment]
@JoviDeCroock Should I transfer this back?

@nmocruz It'd really help in that case to cut down your app to the minimal reproduction. It doesn't make much sense to have `preact-cli` installed (and scripts using Preact-CLI) when you're not actually using it.

**Comment by nmocruz** [comment]
@rschristian  I used the cli just to generate a demo, and then I adjusted the demo to use webpack 5, ts-loader and instead to be near my other project. My other project is using dotnet core, maybe hard to setup and run, but I was able to reprocude it like this

**Comment by rschristian** [comment]
That's totally fine, but you can probably imagine how maintainers might get confused? You say you use `npx webpack serve`, and yet your scripts don't show that at all:

```json
"scripts": {
    "build": "preact build",
    "serve": "SET NODE_OPTIONS=--openssl-legacy-provider sirv build --cors --single",
    "dev": "preact watch",
    "lint": "eslint src",
    "test": "jest"
  },
```

The ReadMe in your project says `npm run build` is how to generate a build: https://github.com/nmocruz/preactjsdemo#cli-commands

It'd help us track down your problem if you removed these things and provided clear instructions alongside a minimal reproduction.

**Comment by nmocruz** [comment]
yes, as I said I used the cli to build a demo to show you the issue. I could remove all cli references on the original code generated by the cli or maybe  have started one from scratch. but I fully understand the read and the scripts should have been removed to not mislead you. I have going to updated the demo, so no one else gets confused
